docs/operator-manual/rbac.md

```diff
@@ -1,195 +1,286 @@
+user management system and has only one built-in user, `admin`. The `admin` user is a superuser and
 it has unrestricted access to the system. RBAC requires [SSO configuration](user-management/index.md) or [one or more local users setup](user-management/index.md).
 Once SSO or local users are configured, additional RBAC roles can be defined, and SSO groups or local users can then be mapped to roles.
 
+There are two main components where RBAC permission can be defined:
+
+- The global RBAC config map (see [argo-rbac-cm.yaml](argocd-rbac-cm-yaml.md))
+- The [AppProject's roles](../user-guide/projects.md#project-roles)
+
 ## Basic Built-in Roles
 
 Argo CD has two pre-defined roles but RBAC configuration allows defining roles and groups (see below).
 
-* `role:readonly` - read-only access to all resources
-* `role:admin` - unrestricted access to all resources
+- `role:readonly`: read-only access to all resources
+- `role:admin`: unrestricted access to all resources
 
 These default built-in role definitions can be seen in [builtin-policy.csv](https://github.com/argoproj/argo-cd/blob/master/assets/builtin-policy.csv)
 
-### RBAC Permission Structure
+## Default Policy for Authenticated Users
+
+When a user is authenticated in Argo CD, it will be granted the role specified in `policy.default`.
+
+!!! warning "Denying Default Permissions"
+
+    **All authenticated users get _at least_ the permissions granted by the default policy. This access cannot be blocked
+    by a `deny` rule.** It is recommended to create a new `role:authenticated` with the minimum set of permissions possible,
+    then grant permissions to individual roles as needed.
+
+## Anonymous Access
+
+Enabling anonymous access to the Argo CD instance allows users to assume the default role permissions specified by `policy.default` **without being authenticated**.
+
+The anonymous access to Argo CD can be enabled using the `users.anonymous.enabled` field in `argocd-cm` (see [argocd-cm.yaml](argocd-cm-yaml.md)).
+
+!!! warning
+
+    When enabling anonymous access, consider creating a new default role and assigning it to the default policy
+    with `policy.default: role:unauthenticated`.
+
+## RBAC Policy Structure
+
+The policy syntax is based on [Casbin](https://casbin.org/docs/overview). There are two different types of policy syntax: one for assigning permissions, and another one for assigning users to internal roles.
+
+**Group**: Allows to assign authenticated users/groups to internal roles.
+
+Syntax: `g, <user/group>, <role>`
+
+- `<user/group>`: The entity to whom the role will be assigned. It can be a local user or a user authenticated with SSO.
+  When SSO is used, the `user` will be based on the `sub` claims, while the group is one of the values returned by the `scopes` configuration.
+- `<role>`: The internal role to which the entity will be assigned.
+
+**Permission**: Allows to assign permissions to an entity.
```

Comment:
What is confusing is that there are 2 things that are a policy: the "policy line" and the policy csv as a whole. When people read the ArgoCD documentation, chances are very low that they will know Casbin. Using the same terminology for these 2 concepts makes the ArgoCD documentation very confusing in my opinion. Even in Casbin, the term "policy" represent both a group of policy and the policy line. 

Using permissions term as the "policy line", as clearly defined here, and policy as the "group of permissions" makes the documentation a lot less confusing in my opinion.

Reply:
The problem is that this approach tries to address one confusion creating another one. I do think that we should call the lines with `p` policy because this is what they are as defined by Casbin. The list of policies can be called in different ways. Suggestions that are already in place:
- Policies (with an `s`)
- RBAC definitions
- Argo CD RBAC